### PR TITLE
cram: new port; meson, py-pyvcf: update

### PR DESCRIPTION
**Pull request by SoapZA**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G65
Xcode 10.0 10A255

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/cram/Portfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                cram
+version             0.7
+categories          devel python
+platforms           darwin
+license             GPL-2+
+maintainers         @SoapZA
```
Comment (mojca) [inline devel/cram/Portfile]: Please add an obfuscated version of your email and ideally `openmaintainer` as well unless there is a good reason for not doing so (but explain in a few words).

File: devel/cram/Portfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+homepage            https://bitheap.org/cram/
+
+master_sites        pypi:c/${name}
+distname            ${name}-${version}
+
+checksums           rmd160  50e5389108496fec6904ccc3e911399b07b4d480 \
+                    sha256  7da7445af2ce15b90aad5ec4792f857cef5786d71f14377e9eb994d8b8337f2f \
+                    size    33527
+
+python.versions         27 34 35 36 37
```
Comment (mojca) [inline devel/cram/Portfile]: We'll start phasing out python 3.4 subports soon (potentially striving towards supporting just 3.6 and 3.7, not sure). Unless there's some strong reason for keeping support for 3.4, it's better to leave it out.

File: devel/meson/Portfile [hunk 3]
```diff
@@ -2,17 +2,15 @@
...
-github.setup        mesonbuild meson 0.47.2
-github.tarball_from releases
-license             Apache-2
+name                meson
+version             0.48.0
 categories          devel python
-maintainers         nomaintainer
 platforms           darwin
+license             Apache-2
+maintainers         @SoapZA
```
Comment (mojca) [inline devel/meson/Portfile]: Same comment about maintainer as above (as well as for the other two ports).

File: devel/meson/Portfile [hunk 4]
```diff
@@ -21,9 +19,14 @@ long_description    Meson  is  a  build system designed to optimize programmer p
                     Valgrind,  CCache  and  the like. It is both extremely fast, and, even more importantly, \
                     as user friendly as possible.
 
-checksums           rmd160  61f9437b9c405be9d6dd4fe53d2ad5657fa14909 \
-                    sha256  92d8afd921751261e36151643464efd3394162f69efbe8cd53e0a66b1cf395eb \
-                    size    1267981
+homepage            https://mesonbuild.com/
+
+master_sites        pypi:m/${name}
```
Comment (mojca) [inline devel/meson/Portfile]: What was wrong with github?

Anyway, this port is a mixture of update, switch to pypi (ok, it makes sense if the two are done together) and unrelated rearrangement of code with no functional changes. It's not absolutely necessary, but we try to decouple the "whitespace" changes from functional changes into two separate commits. But before that I'm really curious what the benefits of pypi would be in the first place

File: devel/cram/Portfile [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                cram
+version             0.7
+categories          devel python
+platforms           darwin
+license             GPL-2+
+maintainers         users.noreply.github.com:SoapZA openmaintainer
```
Comment (pmetzger) [inline devel/cram/Portfile]: We currently require that you include your github handle. Look for examples in other ports, you will need to use that format. Also, is users.noreply.github.com:SoapZA actually valid for you?

File: devel/meson/Portfile [hunk 6]
```diff
@@ -2,17 +2,15 @@
...
-github.setup        mesonbuild meson 0.47.2
-github.tarball_from releases
-license             Apache-2
+name                meson
+version             0.48.0
 categories          devel python
-maintainers         nomaintainer
 platforms           darwin
+license             Apache-2
+maintainers         users.noreply.github.com:SoapZA openmaintainer
```
Comment (pmetzger) [inline devel/meson/Portfile]: See my comment about "maintainers" from earlier.

File: python/py-pyvcf/Portfile [hunk 7]
```diff
@@ -9,7 +9,7 @@ version             0.6.8
 categories-append   science
 platforms           darwin
 license             MIT BSD
-maintainers         @SoapZA
+maintainers         users.noreply.github.com:SoapZA openmaintainer
```
Comment (pmetzger) [inline python/py-pyvcf/Portfile]: See my comment about maintainers from earlier.

**Comment by mojca** [comment]
I'm sorry, yes, I misspelled the port name. Thanks for the fix.

**Comment by SoapZA** [comment]
All email addresses updated

**Comment by mojca** [comment]
Thanks for fixing the maintainers. I'm **personally** not too thrilled about pypi, but it doesn't really matter, we provide ports of both flavours, I leave the decision to others. In any case it's usually maintainer's decision what to use anyway.

The slight advantage of github is that in case of some pre-release patches which are of critical importance to us, it's much easier to temporarily switch to a newer commit (instead of patching), either just for quick testing, or for real. (Until not so long ago meson was completely broken for us and at that point it was helpful to be able to test the changes quickly.)

The rest looks OK (and even pypi is ok if others agree).

**Comment by mojca** [comment]
Changing the source of tarballs from pypi to github means different checksums. I'll fix it and merge this.

**Pull request merged**